.github/workflows/ci.yml

```diff
@@ -78,50 +77,53 @@ jobs:
-    # NOTE(robinson) - This is a fallback in case the lint job does not find the cache.
-    # We can take this out when we implement the fix in CORE-99
-    - name: Setup virtual environment (no cache hit)
-      if: steps.virtualenv-cache.outputs.cache-hit != 'true'
-      run: |
-        python${{ matrix.python-version}} -m venv .venv
-        source .venv/bin/activate
-        make install-ci
-    - name: Test
-      run: |
-        source .venv/bin/activate
-        make install-detectron2
-        sudo apt-get install -y libmagic-dev poppler-utils tesseract-ocr libreoffice
-        make test
-        make check-coverage
-        make install-ingest-s3
-        make install-ingest-github
-        make install-ingest-wikipedia
-        ./test_unstructured_ingest/test-ingest.sh
+      - uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        id: virtualenv-cache
+        with:
+          path: |
+            .venv
+            nltk_data
+          key: unstructured-${{ runner.os }}-${{ matrix.python-version }}-${{ hashFiles('requirements/*.txt') }}
+      # NOTE(robinson) - This is a fallback in case the lint job does not find the cache.
+      # We can take this out when we implement the fix in CORE-99
+      - name: Setup virtual environment (no cache hit)
+        if: steps.virtualenv-cache.outputs.cache-hit != 'true'
+        run: |
+          python${{ matrix.python-version}} -m venv .venv
+          source .venv/bin/activate
+          make install-ci
+      # NOTE: we need to add `make install-ingest-...` for each connector, and some are currently missing
+      # because the tests have not been created e.g. gcs or azure (abs), once a public remote path is available
+      # for testing purposes, the tests can be developed and triggered as part of the CI.
+      - name: Test
+        run: |
+          source .venv/bin/activate
+          make install-detectron2
+          sudo apt-get install -y libmagic-dev poppler-utils tesseract-ocr libreoffice
+          make test
+          make check-coverage
+          make install-ingest-s3
+          make install-ingest-github
+          make install-ingest-wikipedia
+          ./test_unstructured_ingest/test-ingest.sh
 
   changelog:
     runs-on: ubuntu-latest
     steps:
-    - if: github.ref != 'refs/heads/main'
-      uses: dorny/paths-filter@v2
-      id: changes
-      with:
-        filters: |
-          src:
-            - 'unstructured/**'
+      - if: github.ref != 'refs/heads/main'
+        uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          filters: |
+            src:
+              - 'unstructured/**'
 
-    - if: steps.changes.outputs.src == 'true' && github.ref != 'refs/heads/main'
-      uses: dangoslen/changelog-enforcer@v3
+      - if: steps.changes.outputs.src == 'true' && github.ref != 'refs/heads/main'
+        uses: dangoslen/changelog-enforcer@v3
```

Comment:
why all the formatting differences? it makes it hard to find the real diff if there is one (is there?).
same with other .yml file.

Reply:
It's just formatting, as those were not compliant with the standard YAML formatting, mainly inconsistent use of tabs + spaces, as well as some indents missing (nothing changed in terms of functionality). Anyway, I committed those style changes separately so that the difference can be easily seen at https://github.com/Unstructured-IO/unstructured/pull/318/commits/ee58b522fefb047528bfdfc1c036998b15e46e7c

Reply:
OK, I appreciate that! I'd request there be a separate PR for formatting-only yaml changes for the sake of unrelated PR (and main branch commit) diff readability.